### PR TITLE
feat(mojaloop/project#2533): third party deployment guide

### PR DESCRIPTION
**Pull request by lewisdaly**
Adds a third party section to the existing deployment guide

### CODE REVIEW

File: deployment-guide/README.md [hunk 1]
```diff
@@ -342,3 +344,137 @@ Pre-requisites:
 
 - The following postman environment file should be imported or customized as required when running the above listed Postman collections: [Mojaloop-Local-MojaSims.postman_environment.json](https://github.com/mojaloop/postman/blob/master/environments/Mojaloop-Local-MojaSims.postman_environment.json).
 - Ensure you download the **latest patch release version** from the [Mojaloop Postman Git Repository Releases](https://github.com/mojaloop/postman/releases). For example if you install Mojaloop v12.0.**X**, ensure that you have the latest Postman collection patch version v12.0.**Y**.
+
+
+### <a id='overlay-services'></a>6. Overlay Services/3PPI
+
+As of release v13.1.0 of [mojaloop/helm](https://github.com/mojaloop/helm), Mojaloop supports the Third Party API,
```
Comment (elnyry-sam-k) [inline deployment-guide/README.md:351]:
```suggestion
As of [R.C. v13.1.0](https://github.com/mojaloop/helm/tree/release/v13.1.0) of Mojaloop, Third Party API is supported and will be published with the official release of Mojaloop v13.1.0.
```

File: deployment-guide/README.md [hunk 2]
```diff
@@ -342,3 +344,137 @@ Pre-requisites:
...
+which allows Third Party Payment Initiators (3PPIs) the ability to request an account link from a DFSP and initiate
```
Comment (elnyry-sam-k) [inline deployment-guide/README.md:352]:
```suggestion
This allows Third Party Payment Initiators (3PPIs) the ability to request an account link from a DFSP and initiate
```

**Comment by elnyry-sam-k** [comment]
Thanks @lewisdaly ; a couple of minor edits - otherwise looks good!

File: deployment-guide/README.md (hunk 1 above)
Comment (lewisdaly) [reply deployment-guide/README.md:351]: Ah cool, yeah I didn't know I could link to it already.

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**